v2/index.bs

```diff
@@ -101,7 +101,7 @@ The key words <i>MAY</i>, <i>MUST</i>, <i>MUST NOT</i>, <i>RECOMMENDED</i>, and
 # Terminology # {#terminology}
 
 : <dfn>id</dfn>
-:: A locally unique identifier to address the schema.
+:: A locally unique identifier to address the schema. The identifier MAY include suffix that represent the version of the schema. When it is included, it is RECOMMENDED that version follow [[SCHEMA-VER]].
```

Comment:
I think it's important to leave in the versioning section and give an example of how this can be used, why one would want to use it, etc.

Reply:
I'm not sure I understand. Are you suggesting to leave in the `Versioning Guidelines` section?

Reply:
yes, with guidelines that speak to this note

Reply:
```suggestion
:: A locally unique identifier to address the schema. The identifier MAY include a suffix that represents the version of the schema. When it is included, it is RECOMMENDED that the version follow [[SCHEMA-VER]].
```

Reply:
> Are you suggesting to leave in the `Versioning Guidelines` section?

Yes. To my mind, the idea is not to eliminate versioning entirely; only to eliminate the `version` _property_, and put any version detail as a suffix on the identifier. Suitable suffixes might be a year, a year-month, an incrementing semantic version number, etc. The schema identified by such a version-suffixed identifier is immutable, but there may be multiple version suffixes over time, the schema identified by each of which may differ from the schemas identified by the others.